src/components/views/EditIndex.tsx

```diff
@@ -77,9 +71,18 @@ const EditIndex = () => {
 		}
```

Comment:
There's a flickering of the welcome modal when you are not logged and access to the route directly (try refreshing) With this new design I believe we can remove the whole modal

Reply:
like the above if we want to change the flow we should have design team approval first.